003-readable-read-into.md

```diff
@@ -0,0 +1,99 @@
+| Title  | stream.Readable#readInto()  |
+|--------|-----------------------------|
+| Author | @indutny                    |
+| Status | DRAFT                       |
+| Date   | 2016-06-01                  |
+
+
+## 1. Proposal
+
+It is proposed to add `readInto` method and configuration option to the
+`stream.Readable` class.
+
+```js
+const stream = new stream.Readable({
+  readInto: (buf, offset, length, callback) => {
+  }
+});
+
+// Asynchronously read data into `buf.slice(offset, offset + length)`
+// Invoke continuation with either error (`err`) or number of bytes
+// read (`length`)
+stream.readInto(buf, offset, length, (err, length) => {
```

Comment:
Yeah, agreed. simply passing the additional `buf` and `offset` arguments on should be good enough.